assets/js/blocks/product-collection/inspector-controls/price-range-control/PriceTextField.tsx

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable @wordpress/no-unsafe-wp-apis */
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import {
+	// @ts-expect-error Using experimental features
+	__experimentalNumberControl as NumberControl,
+	// @ts-expect-error Using experimental features
+	__experimentalInputControlPrefixWrapper as InputControlPrefixWrapper,
+} from '@wordpress/components';
+
+interface PriceTextFieldProps {
+	value: number;
+	onChange: ( min: number ) => void;
```

Comment:
Component is used for both: min and max so I think it should be agnostic:

```suggestion
	onChange: ( value: number ) => void;
```